locales/en/app.ftl

```diff
@@ -868,7 +868,7 @@ ad-unit-6-before-you-complete = Before you complete that next signup, use an ema
 
 ## Search Engine Optimization
 
-meta-desc = Find out if you’ve been part of a data breach with { -brand-fx-monitor }. Sign up for alerts about future breaches and get tips to keep your accounts safe.
+meta-desc-2 = Find out if you've been part of a data breach with { -brand-fx-monitor }. We’ll help you understand what to do next and continuously monitor for any new breaches.
```

Comment:
nit: one of the smart quotes, `you've`, turned unsmart (yet the `We’ll` is smart).

Reply:
Fixed https://github.com/mozilla/blurts-server/pull/3009/commits/2ea20ca3c11396fe94cce734c830e27d3da6cc10.